modules/extensions-spring-support/src/main/resources/META-INF/mule-extension.xsd

```diff
@@ -0,0 +1,113 @@
+        <xsd:annotation>
+            <xsd:documentation>
+                Application level configuration about how to handle Mule Extensions
+            </xsd:documentation>
+        </xsd:annotation>
+    </xsd:element>
+
+    <xsd:complexType name="extensionsConfigType">
+        <xsd:complexContent>
+            <xsd:extension base="mule:abstractConfigurationExtensionType">
+                <xsd:sequence>
+                    <xsd:choice>
+                        <xsd:element name="dynamic-configuration-expiration" type="dynamicConfigurationExpirationType" minOccurs="0" maxOccurs="1">
+                            <xsd:annotation>
+                                <xsd:documentation>
+                                    Configuration about how often to check for expirable dynamic configuration instances and actually do
+                                    dispose them
+                                </xsd:documentation>
+                            </xsd:annotation>
+                        </xsd:element>
+                    </xsd:choice>
+                </xsd:sequence>
+            </xsd:extension>
+        </xsd:complexContent>
+    </xsd:complexType>
+
+    <xsd:complexType name="dynamicConfigurationExpirationType">
+        <xsd:attribute name="frequency" type="mule:substitutableLong" use="required">
+            <xsd:annotation>
+                <xsd:documentation>
+                    A scalar time value for how often should the platform check for expirable dynamic configs
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+        <xsd:attribute name="timeUnit" type="mule:timeUnitType" use="required">
+            <xsd:annotation>
+                <xsd:documentation>
+                    A time unit that qualifies the frequency attribute
+                </xsd:documentation>
+            </xsd:annotation>
+        </xsd:attribute>
+    </xsd:complexType>
+
+    <xsd:element name="dynamic-config-policy" type="dynamicConfigPolicyType">
+        <xsd:annotation>
+            <xsd:documentation>
+                Settings which define how the platform should manage dynamic instances of a configuration element.
+            </xsd:documentation>
+        </xsd:annotation>
+    </xsd:element>
+
+    <xsd:complexType name="dynamicConfigPolicyType">
+        <xsd:annotation>
+            <xsd:documentation>
+                Settings which define how the platform should manage dynamic instances of a configuration element.
+            </xsd:documentation>
+        </xsd:annotation>
+        <xsd:sequence>
+            <xsd:choice>
+                <xsd:element name="expiration-policy" type="expirationPolicyType" minOccurs="0" maxOccurs="1">
+                    <xsd:annotation>
+                        <xsd:documentation>
+                            Configures the maximum amount of time that a dynamic configuration instance can remain idle
```

Comment:
My comment was related to the fact that the configuration is not expired when this max is reached, because it depends on when the reaper thread is executed. That is why to me this is a minimum :)